dashboard/test/controllers/api/v1/pd/course_facilitators_controller_test.rb

```diff
@@ -1,27 +1,9 @@
 require 'test_helper'
 
 class Api::V1::Pd::CourseFacilitatorsControllerTest < ::ActionController::TestCase
-  test 'admins can see course facilitators' do
-    sign_in create(:admin)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :success
-  end
+  generate_user_tests_for :index, user: [:admin, :workshop_organizer],
+    response: :success, params: {course: Pd::Workshop::COURSES.first}
```

Comment:
nit: Our style guide would suggest that this should be
```
  generate_user_tests_for(
    :index,
    user: [:admin, :workshop_organizer],
    response: :success
    params: {course: Pd::Workshop::COURSES.first}
  )
```
though I'm willing to overlook it.